services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -93,16 +125,21 @@ export const renderDocumentLinkColumn = (
     dataIndex,
     key: dataIndex,
     render: (text = "", record: any) => {
+      const recordWithArchiveIndicator = { ...record, showArchiveIndicator };
       const link = (
-        <div key={record.key ?? record[docManGuidIndex]} title={title}>
+        <div
+          key={record.key ?? record[docManGuidIndex]}
+          className={record.number_of_versions !== undefined ? "file-name-text" : ""}
+          style={record?.number_of_versions === 0 ? { marginLeft: "38px" } : {}}
+        >
           <DocumentLink
             documentManagerGuid={record[docManGuidIndex]}
-            documentName={text}
+            documentName={record.document_name ?? text}
```

Comment:
This should be completely reliable as "text", should be no need for looking at document_name.
The contents of text in column render function is determined by antd internally by record[dataIndex], so just need to pass in the right attribute name to the function.

Reply:
Ah I see, this has been reverted back to `text ` now